openy.info.yml

```diff
@@ -182,6 +182,7 @@ dependencies:
   - openy_prgf_4c
   - openy_prgf_amenities
   - openy_prgf_lto
+  - openy_node_social_post
```

Comment:
ensure you enabled this module during an upgrade of the site in openy.install hook_update_N

Reply:
added